core/src/main/java/com/graphhopper/routing/util/WheelchairFlagEncoder.java

```diff
@@ -0,0 +1,206 @@
+/*
+ *  Licensed to GraphHopper GmbH under one or more contributor
+ *  license agreements. See the NOTICE file distributed with this work for
+ *  additional information regarding copyright ownership.
+ *
+ *  GraphHopper GmbH licenses this file to you under the Apache License,
+ *  Version 2.0 (the "License"); you may not use this file except in
+ *  compliance with the License. You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+package com.graphhopper.routing.util;
+
+import com.graphhopper.reader.ReaderWay;
+import com.graphhopper.storage.IntsRef;
+import com.graphhopper.util.PMap;
+
+import java.util.*;
+
+import static com.graphhopper.routing.util.PriorityCode.*;
+
+/**
+ * A flag encoder for wheelchairs.
+ * <p>
+ *
+ * @author don-philipe
+ */
+public class WheelchairFlagEncoder extends FootFlagEncoder {
+    static final int SLOW_SPEED = 2;
+    static final int MEAN_SPEED = 5;
+    static final int FERRY_SPEED = 15;
+    final Set<String> safeHighwayTags = new HashSet<>();
+    final Set<String> allowedHighwayTags = new HashSet<>();
+    final Set<String> avoidHighwayTags = new HashSet<>();
+    final Set<String> avoidSurfaces = new HashSet<>();
+    final Set<String> avoidSmoothness = new HashSet<>();
```

Comment:
The following variables are already defined in the super class: safeHighwayTags, allowedHighwayTags, ...

Also the static variables should not be overwritten and as they are identical you should be able to just remove them.